src/styles/StudentPortal.css

```diff
@@ -0,0 +1,39 @@
+.btn {
```

Comment:
If someone directly hits this page without login, they should be redirected to the login page.